sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1-basic-operations.cpp

```diff
@@ -2,11 +2,11 @@
 // SPDX-License-Identifier: MIT
 
 /**
- * @brief This sample provides the code implementation to use the Key Vault Secrets SDK client for
+ * @brief This sample provides the code implementation to use the KeyVault Secrets SDK client for
```

Comment:
We should follow `Key Vault` for any documentation as that's how the official documentation is:
https://docs.microsoft.com/en-us/azure/key-vault/

We can join it on the source, but comments/readme/docs should be `Key Vault`

Reply:
done